### ISSUE TITLE
Document menu highlight color is not friendly to people with weak colors

### ISSUE THREAD

**Issue by janryWang** [gatsbyjs/gatsby#16156]
![image](https://user-images.githubusercontent.com/4060976/61997257-98387100-b0d1-11e9-8d0a-cbe5338c01cf.png)

Because I am a weak person, I am very tired to read the document. Please adjust the color depth.

**Comment by lannonbr** [comment]
Can you explain what you mean? I checked the color contrast for the default black in the sidebar and it is 12.62 which meets WCAG AAA standards, and the purple is 5.67 which meets WCAG AA.

**Comment by lannonbr** [comment]
Do you happen to mean that the background color for when you focus / hover over an item does not have a good contrast compared to the rest of the background for items that are not focused?

**Comment by janryWang** [comment]
> Do you happen to mean that the background color for when you focus / hover over an item does not have a good contrast compared to the rest of the background for items that are not focused?

No , that's what I means.
